docs/pages/get-started/installation.mdx

```diff
@@ -8,7 +8,7 @@ To develop applications with Expo, you need two tools. A command-line tool calle
 and a mobile client app called [Expo Go](#2-expo-go-app-for-ios-and) to open the project on iOS and Android platforms.
 Additionally, you can use any web browser to run the project on the web.
 
-> **info** You don't need macOS to build an iOS app with Expo. You only need an iOS device to run the Expo Go app. Windows, Linux, and macOS are all supported for your development machine.
+> **info** You don't need macOS to build an iOS app with EAS. You only need an iOS device to run development builds. Windows, Linux, and macOS are all supported for your development machine.
```

Comment:
I think the wording is a bit confusing here, considering that this is the first page/doc in a user's journey. We are not providing enough context as to what EAS and development builds are. I think we should add links to EAS and development builds here. Will create a follow up PR.